scripts/buildkite/release/push-release.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

? 🤔 This should give it a chance to use `bash` from the nix store, making the script more reproducible across platforms. I'm not entirely sure myself, though.